client/src/Overview/components/StyleSelector.jsx

```diff
@@ -2,21 +2,41 @@ import React from 'react';
 
 const StyleSelector = (props) => {
   if (props.state.styles.length > 0) {
-    let key = -1;
+    let keyOne = -1;
+    let keyTwo = -1;
 
   return (
-    <div className = "styleSelector">
+    <div className = "style-selector">
       <p>STYLE > {props.state.styles[props.OverviewState.styleIndex].name}</p>
       <table>
         <tbody>
         <tr>
            {props.state.styles.map((style)=> {
-             key += 1;
-          return (
-            <td key = {key} onClick = {props.changeStyle}>
-            <img value = {key} src={style.photos[0].thumbnail_url} width="50" height="50"></img>
+             keyOne += 1;
+             if (keyOne >= 4) {
+               return;
+             } else {
+               return (
+                 <td key = {keyOne} onClick = {props.changeStyle}>
+            <img value = {keyOne} src={style.photos[0].thumbnail_url} className = "style-selector-image"></img>
             </td>
             )
+          }
+        })
+       }
+      </tr>
+      <tr>
+           {props.state.styles.map((style)=> {
+             keyTwo += 1;
+             if (keyTwo < 4) {
+               return;
+             } else {
+               return (
+            <td key = {keyTwo} onClick = {props.changeStyle}>
+            <img value = {keyTwo} src={style.photos[0].thumbnail_url} className = "style-selector-image"></img>
```

Comment:
You might be handling this elsewhere, but be aware that most products have a default style. I've been using the photo for the default style as my photo because the default style is also tied to price. You might be accounting for this elsewhere, but if not, definitely check out the difference in price/photos in various styles, as well as the 'default?' property

Reply:
Yea this is kind of a round about way of doing it. I should really rename the keyTwo and keyOne properties to "styleIndex" because thats what they are. Iterating through the style array the key just keeps track of the index.